examples/jsm/renderers/webgl/WebGLBackend.js

```diff
@@ -386,7 +386,20 @@ class WebGLBackend extends Backend {
 		else if ( object.isLineSegments ) mode = gl.LINES;
 		else if ( object.isLine ) mode = gl.LINE_STRIP;
 		else if ( object.isLineLoop ) mode = gl.LINE_LOOP;
-		else mode = gl.TRIANGLES;
+		else {
+
+			if ( material.wireframe === true ) {
```

Comment:
Don't we need a wireframe attribute similar to `WebGLRenderer`?

https://github.com/mrdoob/three.js/blob/69e8505887d7c0c847515bcea2114a4ab8040c36/src/renderers/WebGLRenderer.js#L795-L797

Reply:
Not sure about this one. If that's ok I would like to improve this part in another PR.

Reply:
`WebGPURenderer` handle it in `Geometries.js` and provide the attribute for both backends. We must use `renderObject.getIndex()` to obtain the corresponding indexes.